src/cms-content/vaccines/state-vaccine-phases.json

```diff
@@ -0,0 +1,20 @@
+{
+  "regions": [
+    {
+      "emailAlertVersion": 0,
+      "eligibilityInfoUrl": "https://covidvaccine.mo.gov/",
+      "fips": "29",
+      "locationName": "Missouri",
+      "eligibilityPhases": [
+        {
+          "eligibilityStartDate": "2020-12-14T18:54:06.354Z",
```

Comment:
nit: I think that `vaccinationPhases` and `startDate` are better names, but feel free to ignore/punt